src/com/lilithsthrone/game/inventory/item/AbstractItem.java

```diff
@@ -282,6 +283,11 @@ public boolean isGift() {
 		return itemType.isGift();
 	}
 
+	public boolean isFertilityPill() {
+		return this.getItemType().equals(ItemType.getItemTypeFromId("innoxia_pills_fertility"))
+				|| this.getItemType().equals(ItemType.getItemTypeFromId("innoxia_pills_broodmother"));
+	}
+
```

Comment:
I wonder if a more generic `item.isTypeOneOf(String itemType...)` would be better. More useful for modders, I think.

Reply:
Done